diff --git a/third_party/spirv-tools/utils/update_build_version.py b/third_party/spirv-tools/utils/update_build_version.py
index 5a78ada2..29f67b32 100755
--- a/third_party/spirv-tools/utils/update_build_version.py
+++ b/third_party/spirv-tools/utils/update_build_version.py
@@ -111,7 +111,7 @@ def describe(repo_path):
     Runs 'git describe', or alternately 'git rev-parse HEAD', in directory.  If
     successful, returns the output; otherwise returns 'unknown hash, <date>'."""
 
-    success, output = command_output(['git', 'describe'], repo_path)
+    success, output = command_output(['git', 'describe', '--tags'], repo_path)
     if not success:
       output = command_output(['git', 'rev-parse', 'HEAD'], repo_path)
 
